conda/mpich/meta.yaml

```diff
@@ -67,6 +61,12 @@ requirements:
     - {{ moose_xorg_x11 }}      # [linux]
     - {{ moose_libglu }}        # [linux]
     - {{ moose_mesalib }}       # [linux]
+    - cmake
+    - make
+    - libtool
+    - autoconf
+    - automake 1.16.1
+    - m4
```

Comment:
I just moved this around for visual appearance in the file.

Reply:
✨ 👀